modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_pdf_generator.rb

```diff
@@ -54,7 +54,8 @@ def perform(claim_id, middle_initial) # rubocop:disable Metrics/MethodLength
         start_docker_container_job(auto_claim&.id) if auto_claim.status != errored_state_value
       rescue Faraday::ParsingError, Faraday::TimeoutError => e
         set_errored_state_on_claim(auto_claim)
-        error_message = get_error_message(e)
+        error_message = auto_claim.evss_response&.dig(0, 'text') if auto_claim.evss_response.present?
```

Comment:
The `set_errored_state` method call is still included in these (line 56 and 67), is that intentional?  Only asking because it was removed in the docker_container job so seemed like this would be in line with that as well, but I could be missing some part of sync the flow too?